src/components/Input/FileInput.tsx

```diff
@@ -5,11 +5,12 @@ import { ButtonVariant } from '../Button/Button'
 
 export interface FileInputProps {
   text?: string
-  onUpload?: (file: File) => void
+  onUpload?: (files: File[]) => void
```

Comment:
`<FileInput />` component is used in BankTransactions-related components, like `BusinessForm`, where functions expect parameter to be a single file (`File`), and not a list of files (`File[]`). I would opt to update functions there with sth like this:

```tsx
onUpload={files => receiptsRef.current?.uploadReceipt(files[0])}
```